sdk/quantum/quantum-jobs/package.json

```
@@ -6,79 +6,118 @@
+    "sinon": "^9.0.2",
+    "ts-node": "^8.3.0",
+    "typescript": "4.1.2",
     "uglify-js": "^3.4.9",
-    "@microsoft/api-extractor": "7.7.11",
-    "mkdirp": "^1.0.4"
+    "cross-env": "^7.0.2",
+    "typedoc": "0.15.2"
   },
   "homepage": "https://github.com/Azure/azure-sdk-for-js",
-  "repository": {
-    "type": "git",
-    "url": "https://github.com/Azure/azure-sdk-for-js.git"
-  },
+  "repository": "github:Azure/azure-sdk-for-js",
   "bugs": {
     "url": "https://github.com/Azure/azure-sdk-for-js/issues"
   },
   "files": [
+    "dist-esm/**/*.js",
+    "dist-esm/**/*.js.map",
+    "dist-esm/**/*.d.ts",
+    "dist-esm/**/*.d.ts.map",
     "dist/**/*.js",
     "dist/**/*.js.map",
     "dist/**/*.d.ts",
     "dist/**/*.d.ts.map",
-    "esm/**/*.js",
-    "esm/**/*.js.map",
-    "esm/**/*.d.ts",
-    "esm/**/*.d.ts.map",
-    "./src//**/*.ts",
     "README.md",
     "rollup.config.js",
     "tsconfig.json"
   ],
   "scripts": {
-    "build": "tsc && rollup -c rollup.config.js && mkdirp ./review &&  npm run extract-api",
+    "build": "npm run build:node && npm run build:browser",
+    "build:node": "tsc -p . && cross-env ONLY_NODE=true rollup -c 2>&1 && npm run extract-api",
+    "build:browser": "tsc -p . && cross-env ONLY_BROWSER=true rollup -c 2>&1",
+    "build:test": "npm run build:test:node && npm run build:test:browser",
+    "build:test:node": "tsc -p . && cross-env ONLY_NODE=true rollup -c rollup.test.config.js 2>&1",
+    "build:test:browser": "tsc -p . && cross-env ONLY_BROWSER=true rollup -c rollup.test.config.js 2>&1",
+    "check-format": "prettier --list-different --config ../../../.prettierrc.json --ignore-path ../../../.prettierignore \"src/**/*.ts\" \"test/**/*.ts\" \"*.{js,json}\"",
     "clean": "rimraf dist dist-esm dist-browser test-dist test-browser types *.tgz *.log",
-    "extract-api": "api-extractor run --local",
+    "coverage": "nyc --reporter=lcov --exclude-after-remap=false mocha -t 120000 test-dist/index.js --reporter ../../../common/tools/mocha-multi-reporter.js",
+    "extract-api": "tsc -p . && api-extractor run --local",
+    "format": "prettier --write --config ../../../.prettierrc.json --ignore-path ../../../.prettierignore \"src/**/*.ts\" \"test/**/*.ts\" \"*.{js,json}\"",
+    "lint:fix": "eslint package.json api-extractor.json src test --ext .ts --fix",
+    "lint": "eslint package.json api-extractor.json src test --ext .ts -f html -o quantum-jobs-lintReport.html || exit 0",
+    "prebuild": "npm run clean",
+    "pack": "npm pack 2>&1",
+    "integration-test": "npm run integration-test:node && npm run integration-test:browser",
+    "integration-test:browser": "npm run build:test:browser && cross-env TEST_MODE=live karma start --single-run",
+    "integration-test:node": "npm run build:test:node && nyc mocha -r esm --require source-map-support/register --reporter ../../../common/tools/mocha-multi-reporter.js --timeout 180000 --full-trace dist-esm/test/*.spec.js dist-esm/test/**/*.spec.js",
```

Same as my comment at https://github.com/Azure/azure-sdk-for-js/pull/13385/files#r564095574
```suggestion
    "integration-test:node": "nyc mocha -r esm --require source-map-support/register --reporter ../../../common/tools/mocha-multi-reporter.js --timeout 180000 --full-trace dist-esm/test/*.spec.js dist-esm/test/**/*.spec.js",
```